superbench/benchmarks/micro_benchmarks/_export_torch_to_onnx.py

```diff
@@ -0,0 +1,188 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+"""Export PyTorch models to ONNX format."""
```

Comment:
This looks like a utility module, why and underline at the beginning?

Reply:
a utility for internal use in micro_benchmarks